_includes/ont_runs_add_pools.md

```diff
@@ -0,0 +1,16 @@
+## 5.3 Adding pools to runs
+ 
+The Run (representing an instrument run) is associated with Pools using a
+_Sequencing Container_.
+
+1. On the Edit Run page, scroll down to the _Flow Cells_ section.
+1. Select the only flow cell (Number 1).
+1. Click _Assign Pool_ from the toolbar.
+1. Search for the pool you previously created. Try _Outstanding Orders (Matched Chemistry)_.
```

Comment:
I saw the instruction to 'Search for the pool' and was confused when the page didn't have a search box. I then saw the item just called "Search" in the list so I clicked that. That led me to typing 'Outstanding Orders (Matched Chemistry)' into the search box as per the rest of the instruction, getting nothing and thinking 'how would I have known to type that anyway?'. This instruction may need re-wording.